pkgs/applications/office/gnumeric/default.nix

```diff
@@ -14,9 +14,19 @@ in stdenv.mkDerivation rec {
     sha256 = "0c8dl1kvnj3g32qy3s92qpqpqfy0in59cx005gjvvzsflahav61h";
   };
 
+  patches = stdenv.lib.optional stdenv.isDarwin
+    # https://gitlab.gnome.org/GNOME/gnumeric/issues/402
+    (fetchurl {
+      name = "math-gamma.patch";
+      url = "https://gitlab.gnome.org/GNOME/gnumeric/uploads"
```

Comment:
Can we stop splitting URLs? It makes them much harder to open them.

Reply:
OK.  I've always been borderline on this.  This particular case isn't too long, but generally I'm trying to keep lines short (80&ndash;100), as otherwise they tend not to play too nice e.g. with 3-way merge tools.

Reply:
IMHO, for URLs, it is still better if they are occasionally wrapped/cropped by a tool than if they cannot be clicked.